### PR TITLE
feat: h1 and h2 refactored

### PR DESCRIPTION
**Pull request by GustavoUT22**

### CODE REVIEW

File: src/ui/Titles/H2.tsx [hunk 1]
```diff
@@ -1,14 +1,25 @@
-import { PropsWithChildren } from 'react'
 import styled from 'styled-components'
+import { colors } from 'styles/colors'
```
Comment (Mardecera) [inline src/ui/Titles/H2.tsx:2]: "styles/colors" is not a library, it should be next to "ui/Titles", not next to "styled-components".

File: src/ui/Titles/H1.tsx [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+import styled from 'styled-components'
+
+import { colors } from 'styles/colors'
 
 const StyledH1 = styled.h1`
   font-weight: bold;
   font-size: clamp(2rem, -1rem + 9.6vw, 5rem);
   line-height: 1.5;
+
+  .span-point {
```
Comment (CrhistianT7) [inline src/ui/Titles/H1.tsx:11]: change the name to .dot only

File: src/ui/Titles/H2.tsx [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+
+import { IH2Props } from 'types/ui/Titles'
 
 const StyledH2 = styled.h2`
   font-weight: bold;
   font-size: clamp(2rem, 0.625rem + 4.4vw, 3.375rem);
   line-height: 1.5;
+
+  .span-point {
```
Comment (CrhistianT7) [inline src/ui/Titles/H2.tsx:11]: same here. change the name to .dot only

**Review by CrhistianT7 — APPROVED** [review]

File: src/ui/Titles/H2.tsx (hunk 1 above)
Comment (GustavoUT22) [reply src/ui/Titles/H2.tsx:2]: done

File: src/ui/Titles/H1.tsx (hunk 2 above)
Comment (GustavoUT22) [reply src/ui/Titles/H1.tsx:11]: done

File: src/ui/Titles/H2.tsx (hunk 3 above)
Comment (GustavoUT22) [reply src/ui/Titles/H2.tsx:11]: done

**Pull request merged**